test/tools/unified-spec-runner/match.ts

```diff
@@ -253,7 +253,12 @@ export function specialCheck(
   } else if (isExistsOperator(expected)) {
     // $$exists
     const actualExists = actual !== undefined && actual !== null;
-    expect((expected.$$exists && actualExists) || (!expected.$$exists && !actualExists)).to.be.true;
+    expect(
+      (expected.$$exists && actualExists) || (!expected.$$exists && !actualExists),
+      `expected value at ${path.join('')} to not exist, but received ${JSON.stringify(
```

Comment:
We need a message fix here, and lets use `ejson` tag template

Reply:
We might want to consider improving this assertion. Currently it's checking for nullishness, should it be more exact and actually be asserting for property existence?